### DISCUSSION TITLE
`useTheme` Returns Empty Object or `undefined`

### DISCUSSION THREAD

**Discussion by okerx** [gluestack/gluestack-ui#1078] [Feature Request]
### Description

Since my app is wrapped with `GluestackUIProvider`, I expect to get the `theme` object using the hook `useTheme`. instead, it returns empty object `{}`. accessing the theme object is necessary if I wanna style non-gluestack components (e.g. FontAwesome icons or React Navigation components)

### CodeSandbox/Snack link

https://snack.expo.dev/@ammaroker/frisky-mixed-nuts

### Steps to reproduce

1. Bootstrap a new gluestack-ui project.
2. Wrap your app with `GluestackUIProvider`
3. Try to access the `theme` object within a component wrapped inside `GluestackUIProvider `
4. the `theme` object will be an empty object.

### gluestack-ui Version

0.2.41

### Platform

- [X] Expo
- [X] React Native CLI
- [ ] Next
- [X] Web
- [X] Android
- [X] iOS

### Other Platform

N/A

### Additional Information

My main goal here is to access the theme styles without the text-based variable. This will make it possible to integrate this library with others.
cheers,

**Comment by Viraj-10** [comment]
Hey @ammar-oker, `useTheme` is currently internal hook. It is not similar to `Nativebase`. We are adding hooks for accessing config.

**Reply by GollyJer** [reply]
I just came to ask the same question. Is there an alternative recommended way to access the theme colors or are you in the process of implementing something? 

For now, I'm importing my config and accessing directly. `gluestackConfig.tokens.colors`

Thanks!

**Reply by Viraj-10** [reply]
you can use `useToken` hook from @gluestack-ui/themed. you can find docs [here](https://gluestack.io/ui/docs/hooks/use-token).
